composer.json

```diff
@@ -28,21 +28,21 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
+        "php": "^8.0",
```

Comment:
Is this upgrade considered a _breacking change_ (thus we need to bump version to 2.0.0)?

Reply:
@enricobono I don't think so based upon my understanding, referring to this post that sums up how I'm thinking about it https://medium.com/@sampart/semantic-versioning-when-you-change-the-required-programming-language-version-16a3a3555c95. 

The short version is: We're not changing the actual API for the package in any breaking way, merely the required PHP version and Composer will handle that if you don't have PHP 8 available on your machine you'll get the last version that does work for you.

Reply:
Make sense, thanks.